docs/developers/dependencies.md

```diff
@@ -0,0 +1,55 @@
+
+---
+title: Dependency Management
+---
+
+You may want your plugin to hook into another. Reinventing the wheel takes too long. Either way, dependencies are common,
+but they need to be managed wisely, lest you fall into dependency hell.
+
+### Plugin dependencies
+
+Adding a dependency on another plugin is done in the `@Plugin` annotation on your main class. Let's revisit that briefly:
```

Comment:
Change to "Adding a dependency on another plugin is done throught the `@Plugin` annotation in your main class."

Reply:
How about "done with"?

Reply:
That'll do as well, just what came to mind